packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -526,14 +528,16 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
   ): JSX.Element {
     const { triggerButton } = this.props
     const content = this.getSelectedItemAsString(this.state.value)
+    const triggerButtonId = triggerButton['id'] || this.defaultTriggerButtonId
 
     const triggerButtonProps = getToggleButtonProps({
       onFocus: this.handleTriggerButtonOrListFocus,
       onBlur: this.handleTriggerButtonBlur,
       onKeyDown: e => {
         this.handleTriggerButtonKeyDown(e, rtl)
       },
-      'aria-label': content,
+      'aria-label': undefined,
```

Comment:
if somebody provides `aria-label` should not we apply it to the trigger or to the input as well? the same for aria-describedby

Reply:
I don't see it as being suggested by ARIA, and if the user wants to do his own magic with label and describedby he can always use the slots.

But if you think we should handle it by default then we can create a new task.